src/libecalc/common/units.py

```diff
@@ -49,6 +49,7 @@ class Unit(str, Enum):
     KG_SM3 = "kg/Sm3"
     KG_M3 = "kg/m3"
     STANDARD_CUBIC_METER = "Sm3"
+    CUBIC_METER = "m3"
```

Comment:
Sm3 is for compressors, m3 is for pumps? Also not Am3?

Reply:
Rate unit is Am3 both for pump- and compressor models (spec in yaml). However, the rate vector in models for compressor is `rate_sm3_day`, while it is  `inlet_liquid_rate_m3_per_day` for pumps. That is why i added CUBIC_METER.

Reply:
Have changed to Sm3/d".